site/checklist_fr.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: page
+title: OpenGates Checklist
+permalink: /checklist/fr
+---
+
+
+Une liste de contrôle de référence pour vous aider à éviter le contrôle d'accès dans vos publications, articles, vidéos, discussions, présentations, podcasts, sites et dans tout autre contenu de développement. 
+
+## Liste de contrôle anti-gardien
+
+La liste de contrôle est divisée en sections pour nous aider à ouvrir les portes et à éviter le contrôle de ces portes.
+
+**Aller à:**
+// A FAIRE: les liens seront bientôt ajoutés
+
+### Inclusion
+ - [ ] Les images doivent varier en fonction de l'âge, des capacités physiques et mentales, du peuple, de la nationnalité, du genre et des types de corps ([ref](https://www.forumone.com/ideas/how-to-choose-diverse-and-inclusive-photos/))
+ - [ ] Les exemples de noms doivent être différents du status ([ref](https://www.christytuckerlearning.com/diverse-characters-in-learning-scenarios/))
+ - [ ] La liste des personnes ne doit pas être homogène ([ref](https://www.forumone.com/ideas/how-to-choose-diverse-and-inclusive-photos/))
+ - [ ] Est-ce que tous ceux qui pourraient voir cela seraient d'accord avec la façon dont ils sont représentés ? ([ref](https://www.forumone.com/ideas/how-to-choose-diverse-and-inclusive-photos/))
+ - [ ] Y a-t-il des stéréotypes perpétués ? ([ref](https://www.forumone.com/ideas/how-to-choose-diverse-and-inclusive-photos/))
+ - [ ] Qui est absent ou exclu ? ([ref](https://www.forumone.com/ideas/how-to-choose-diverse-and-inclusive-photos/))
+ - [ ] item ([ref](A FAIRE))
+
+#### Inclusion de Genre
+ - [ ] Évitez d'utiliser des mots sexués et des expressions familières ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Avez-vous utilisé « homme », ou « hommes » ou des mots les contenant pour désigner des personnes qui ne sont peut-être pas des hommes ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Avez-vous utilisé « il », « lui », « son » ou « lui-même » pour désigner des personnes qui ne sont peut-être pas des hommes ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Si vous avez mentionné le sexe ou le genre de quelqu'un, était-il nécessaire de le faire ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Utilisez-vous des stéréotypes professionnels (ou autres) ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Fournissez-vous les mêmes types d'informations et de descriptions lorsque vous écrivez sur des personnes de genres différents ?([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Utilisez-vous « ils » comme une option neutre en termes de genre ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
```

Comment:
Utilisez-vous « iel » comme une option neutre en termes de genre ? "Iel" is the equivalent of "they", as: a gender neutral pronoun. Note that this is much less commonly used in French (yet?) than in English, but it just entered "Le Robert", a major French dictionnary.

Reply:
Indeed, "iel" has just entered the dictionary and we don't use it that much yet.